src/components/Register/index.jsx

```diff
@@ -0,0 +1,252 @@
+import { Alert, Button, Checkbox, Input } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import { useTranslation } from "react-i18next";
+
+const Register = (props) => {
+  const [t] = useTranslation();
+  const maxLenghtPassword = 8;
+  const [registerInformation, setRegisterInformation] = useState({
+    fullName: "",
+    age: "",
+    email: "",
+    password: "",
+    agree: false,
+  });
+  const [errors, setErrors] = useState({});
+  const [termsOpen, setTermsOpen] = useState(false);
```

Comment:
we should start boolean variables with is, should, etc. because 

termsOpen

does it mean: boolean, is terms open or not?

does it mean: open the terms (a verb, sounds like a function)

isTermsOpen => no question.

Reply:
PTAL